### PR TITLE
(PR) Add mobile menu with js

### PR DESCRIPTION
**Pull request by ivanmvh**
Add a mobile menu to the porfolio mobile version using js (main.js)

### CODE REVIEW

**Comment by ivanmvh** [comment]
Adding mobile-menu dynamics using js  to porfolio project.
We implement the following features:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const showMobileMenu = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+const crossDiv = document.querySelector('.close-menu');
+
+// On icon click:
+//  1) Hide the icon
+//  2) Show the mobile-menu  (open class)
```
Comment (sja-thedude) [inline main.js]: - [x] Kindly remove the commented code as per best HTML practices.

File: css/styles.css [hunk 2]
```diff
@@ -96,6 +109,73 @@ nav {
...
+  right: 0;
+  height: 0%;
+  overflow: hidden;
+  background-color: #396df2;
+  opacity: 0.9;
+  z-index: 5;
+}
+
+.blur {
+  filter: blur(8px);
```
Comment (sja-thedude) [inline css/styles.css]: - [x] Kindly make sure to follow the font weight of the menu options. 
- [x] Kindly make sure to follow the same background colour for the mobile menu. Currently, it's #396df2 but it should be #6070FF according to Figma design.
- [x] Kindly make sure to give a blur effect using a backdrop to the mobile menu.
- [x] Kindly make sure to follow the spacing between the menu options as per the Figma design.

Screenshot 2022-10-11 at 11 46 17 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ivanmvh, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ Good job so far

## Required 
- [x] Kindly make sure that you have html-css-js linters. Currently, you are following html-css linters. Remove the current linters and paste html-css-js linters so that js linters also show up.
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (ivanmvh) [reply main.js]: There are comment to clarify tle process the algorithm. They are used in programming as a tool to document.
This an example of zillions:
[https://www.geeksforgeeks.org/javascript-program-to-check-whether-two-strings-are-anagram-of-each-other/](url)

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @ivanmvh,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - No linter errors. ✔️
 - Great UI design. very similar to Figma's. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**